src/MSBuildLocatorAnalyzer.CodeFixes/MSBuildLocatorAnalyzer.CodeFixes.csproj

```diff
@@ -0,0 +1,37 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>net48;netcoreapp3.1</TargetFrameworks>
```

Comment:
This must be netstandard2.0

I missed reviewing this earlier and only looked when you asked for how this is going to be shipped.